src/staff.js

```diff
@@ -0,0 +1,17 @@
+const express = require("express");
+const {
+  authenticationMiddleware,
+  authorizationMiddleware,
+} = require("./middlewares");
+
+const router = express.Router();
+
+router.use(express.json());
```

Comment:
This is probably unnecessary as the JSON middleware has already been added in server.js line 12.